### PR TITLE
**Very small** changes to the smg to make the recoil less intense when you are strafing/rename weapon_mom_smg to weapon_mom_machinegun

### PR DESCRIPTION
**Pull request by CoopJax**
A tiny edit to smg recoil when you're moving because I find it hard to strafe with the big recoil. It might not be needed and that's fine. Thanks alot for reviewing this in advance :p

### Checklist
- [x ] **I have thoroughly tested all of the code I have modified/added/removed to ensure something else did not break**
- [ ] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x ] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [ x] If I have added or modified any visual assets (models, materials, panels, effects, etc), I have taken screenshots / videos of them and attached them to this PR directly (screenshots uploaded through github, videos uploaded to youtube and linked)
[Recoil Demo](https://cdn.discordapp.com/attachments/765655141525225483/765738843423899668/2020-10-13_19-52-03.mp4) (Note that I am shooting the entire time)
- [ ] If I have modified any console command, console variable, or momentum entity, I have opened an issue (or a PR) for it in the [Momentum Mod documentation repository](https://github.com/momentum-mod/docs)
- [x ] My commits are relatively small and scoped to the best of my ability
- [ x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x ] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

**Comment by CoopJax** [comment]
I didn't want this to be opened quite yet... I wanted to edit some files so that the build wouldn't break.

File: mp/src/game/shared/momentum/weapon/weapon_mom_smg.cpp [hunk 1]
```diff
@@ -38,11 +38,11 @@ void CMomentumMachinegun::PrimaryAttack()
 
     // Kick the gun based on the state of the player.
     if (!FBitSet(pPlayer->GetFlags(), FL_ONGROUND))
-        pPlayer->KickBack(0.9, 0.45, 0.35, 0.04, 5.25, 3.5, 4);
+        pPlayer->KickBack(1.0, 0.55, 0.40, 0.04, 6.0, 4.0, 5);
```
Comment (braem) [inline mp/src/game/shared/momentum/weapon/weapon_mom_smg.cpp:41]: More kickback when we're on the ground? Confused by this change.

**Comment by CoopJax** [comment]
Yeah, I was trying to rename it to mom_weapon_machinegun and it broke everything. I'll try that.

**Comment by braem** [comment]
> 
> 
> Yeah, I was trying to rename it to mom_weapon_machinegun and it broke everything. I'll try that.

Also please include a commit summary instead of leaving it blank to avoid getting commits like "Update mom_weapon_smg".

**Comment by CoopJax** [comment]
There still seems to be an error that I can't figure out.

**Comment by CoopJax** [comment]
But I think I got mostly everything

**Comment by CoopJax** [comment]
> Remember to change the `#include` at the top of the files as well, it still points to the old file.
On what files?

**Comment by Gocnak** [comment]
Try to compile the game after every change and before you commit. It helps a lot with finding issues.

**Comment by CoopJax** [comment]
Yeah, I found the error. It's really cryptic and confusing though. Something to do with the build script, not my actual code.

**Comment by Gocnak** [comment]
Well there still is an issue with your code.

When you rename something, you have to make sure there's not leftovers in various files. Try searching for "mom_weapon_smg". Visual studio should also be highlighting things wrong with red.

**Comment by CoopJax** [comment]
Well I tried to find anything with weapon_mom_smg in it and change it. Build is still giving me an error. error

**Comment by Gocnak** [comment]
Ah. Any time you edit any vpc file, or change branch on git, you'll want to rerun `creategameprojects.bat`. Not as admin, just normally. This will generate an updated momentum.sln.

**Comment by CoopJax** [comment]
Ok, I did it. It debugged and ran successfully!

**Comment by CoopJax** [comment]
Everything is working and i'm done here.

File: mp/src/game/client/client_momentum.vpc [hunk 2]
```diff
@@ -242,7 +242,7 @@ $Project "Client (Momentum)"
                 $File "$SRCDIR\game\shared\momentum\weapon\weapon_mom_shotgun.h"
                 $File "$SRCDIR\game\shared\momentum\weapon\weapon_mom_shotgun.cpp"
                 $File "$SRCDIR\game\shared\momentum\weapon\weapon_mom_smg.h"
```
Comment (Gocnak) [inline mp/src/game/client/client_momentum.vpc:244]: Rename the header to match.

**Review by Gocnak — CHANGES_REQUESTED** [review]

File: mp/src/game/client/client_momentum.vpc (hunk 2 above)
Comment (CoopJax) [reply mp/src/game/client/client_momentum.vpc:244]: Oh, forgot about that.

**Comment by CoopJax** [comment]
Header file errors are fixed and renamed.

**Comment by CoopJax** [comment]
Is everything good now? Anything extra I need to do?

**Review by Gocnak — APPROVED** [review]

**Pull request merged**